src/engine/Server.cpp

```diff
@@ -390,6 +435,30 @@ string Server::composeResponseJson(const ParsedQuery& query,
   return j.dump(4);
 }
 
+// _____________________________________________________________________________
+string Server::composeResponseSparqlJson(const ParsedQuery& query,
+                                         const QueryExecutionTree& qet,
+                                         size_t sendMax) const {
+  shared_ptr<const ResultTable> rt = qet.getResult();
+  _requestProcessingTimer.stop();
```

Comment:
Are the two lines above this necessary?
The timer is stopped and then immediately continued, and the forcing of the result computation here which is then never used will also be triggered again in the `writeResultAsSparqlSomething` functions.